tests/Filter/IgnorePathFilterTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * (c) Olivier Laviale <olivier.laviale@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace tests\olvlvl\ComposerAttributeCollector\Filter;
+
+use Composer\IO\IOInterface;
+use olvlvl\ComposerAttributeCollector\Filter\IgnorePathFilter;
+use PHPUnit\Framework\TestCase;
+
+final class IgnorePathFilterTest extends TestCase
+{
+    /**
+     * @dataProvider provideFilterMatching
+     */
+    public function testFilterMatching(string $filepath, string $class, array  $paths): void
+    {
+        $io = $this->createMock(IOInterface::class);
+        $io
+            ->expects($this->never())
+            ->method('debug');
+
+        $filter = new IgnorePathFilter('/app', $paths);
```

Comment:
Better provide `getcwd()` value instead of using a static value, one might want to run tests outside of the container.

```suggestion
        $filter = new IgnorePathFilter($cwd, $paths);
```

Reply:
It is actually irrelevant to what environment it runs. It's a unit test and the value is just something to assert against.
But improved the tests to have it passed from data provider for better flexibility of the tests.
I also changed the value from `/app` to `/path/to/project/root` to better denote that that's what it is.